### ISSUE TITLE
[Question] Does Quantum support to run automation tests on local device?

### ISSUE THREAD

**Issue by SonNH70** [Project-Quantum/Quantum#95]
I did not find any documents for setting up running the automation tests on local device.

**Comment by MoonLikeSinging** [comment]
Yes, but latest version https://github.com/Perfecto-Quantum/Quantum-Starter-Kit always start edge browser.
I use quantum 1.22, it works well on local device.
set property with webdriver.chrome.driver=src/main/resources/common/chromedriver.exe
